gnome-base/gnome-shell/gnome-shell-45_alpha.ebuild

```diff
@@ -0,0 +1,182 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{10..12} )
+
+inherit gnome.org gnome2-utils meson optfeature python-single-r1 virtualx xdg
+
+DESCRIPTION="Provides core UI functions for the GNOME desktop"
+HOMEPAGE="https://wiki.gnome.org/Projects/GnomeShell https://gitlab.gnome.org/GNOME/gnome-shell"
+
+LICENSE="GPL-2+ LGPL-2+"
+SLOT="0"
+IUSE="elogind gtk-doc +ibus +networkmanager systemd test"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}
+	?? ( elogind systemd )"
+RESTRICT="!test? ( test )"
+
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~riscv ~x86"
+
+# libXfixes-5.0 needed for pointer barriers and #include <X11/extensions/Xfixes.h>
+DEPEND="
+	>=gnome-extra/evolution-data-server-3.46.0:=
+	>=app-crypt/gcr-3.90.0:4=[introspection]
+	>=dev-libs/glib-2.68:2
+	>=dev-libs/gobject-introspection-1.49.1:=
+	>=dev-libs/gjs-1.73.1[cairo]
+	>=gui-libs/gtk-4:4[introspection]
+	>=x11-wm/mutter-45_alpha:0/13[introspection,test?]
+	>=sys-auth/polkit-0.120_p20220509[introspection]
+	>=gnome-base/gsettings-desktop-schemas-42_beta[introspection]
+	>=x11-libs/startup-notification-0.11
+	>=app-i18n/ibus-1.5.19
+	>=gnome-base/gnome-desktop-40.0:4
+	networkmanager? (
+		>=net-misc/networkmanager-1.10.4[introspection]
+		net-libs/libnma[introspection]
+		>=app-crypt/libsecret-0.18
+		dev-libs/dbus-glib
+	)
+	systemd? (
+		>=sys-apps/systemd-242:=
+		>=gnome-base/gnome-desktop-3.34.2:3=[systemd]
+	)
+	elogind? ( >=sys-auth/elogind-237 )
+
+	app-arch/gnome-autoar
+	dev-libs/json-glib
+
+	>=app-accessibility/at-spi2-core-2.46:2[introspection]
+	x11-libs/gdk-pixbuf:2[introspection]
+	dev-libs/libxml2:2
+	x11-libs/libX11
+
+	>=media-libs/libpulse-2[glib]
+	dev-libs/libical:=
+	>=x11-libs/libXfixes-5.0
+
+	gui-libs/gtk:4[introspection]
+
+	${PYTHON_DEPS}
+	$(python_gen_cond_dep '
+		dev-python/pygobject:3[${PYTHON_USEDEP}]
+	')
+	media-libs/libglvnd[X]
+	>=sys-apps/systemd-246
```

Comment:
The systemd requirement was raised in https://gitlab.gnome.org/GNOME/gnome-shell/-/commit/51dc50144ecacc9ac1f807dcc6bdf4f1d49343ae

It was just raised, not added. We should update the existing systemd dependency and not add a new (unconditional) one.